app/pods/components/player-csv-leaderboard/component.js

```diff
@@ -0,0 +1,38 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember-decorators/service';
+import { restartableTask } from 'ember-concurrency-decorators';
+const moment = require('moment');
+
+export default class PlayerCsvLeaderboardComponent extends Component {
+
+  @service api;
+
+  didReceiveAttrs() {
+    this.fetchLeaderboardTask.perform();
+  }
+
+  /**
+   * Fetches the leaderboard data for the 
+   * CSV submission and humanizes the timestamp
+   */
+
+  @restartableTask
+  *fetchLeaderboardTask() {
+    let csvId = this.get('csvId');
+
+    // Fetch the leaderboard data from the server
+    this.api.request(`/csvs/${csvId}/leaderboard`, {
+      method: 'GET'
+    }).then(response => {
+      // Humanize the 'createdAt' time
+      for (let i = 0; i < response.length; i++) {
+        response[i].firstname = response[i].firstname.charAt(0).toUpperCase() + response[i].firstname.slice(1).toLowerCase();
```

Comment:
you can simply `response[i].firstname.capitalize()` Instead
Also use `.map()` instead
```